### PR TITLE
#7 top songs page basic server

### PR DESCRIPTION
**Pull request by Toni-Zoaretz**

### CODE REVIEW

File: server/config/config.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+PORT=5000
+NODE_ENV=development
```
Comment (yishain11) [inline server/config/config.env:2]: .env should not be uploaded to source control! we put secrets there.
add gitignore and make sure that this file is included

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import dotenv from "dotenv";
+import express from "express";
+import { fileURLToPath } from "url";
+import path, { dirname } from "path";
```
Comment (yishain11) [inline server/server.js:4]: if you deconstruct "path", you don't neet to import "path", just add "join" to the object

File: server/server.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const app = express();
+
+dotenv.config({ path: path.join(__dirname, "config.env") });
+
+app.get("/", (req, res) => {
```
Comment (yishain11) [inline server/server.js:13]: remove testing enpoint

**Review by yishain11 — CHANGES_REQUESTED** [review]

File: server/config/config.env (hunk 1 above)
Comment (Toni-Zoaretz) [reply server/config/config.env:2]: i already have .gitignor with .env inside I can see it from the beginning. are you sure about it?

**Pull request merged**